### PR TITLE
[MLMD Lineage] Initial Lineage explorer draft implemented

### PR DESCRIPTION
**Pull request by avdaredevil**
## About
This PR addresses adding the Lineage Explorer for MLMD into this UI. This is a P0 ask for Kubeflow Core and Pipelines. There will be follow up changes and updates to polish this change and make it robust. Consider this as a semi-polished blueprint. Thanks to @ajayalfred for the source mocks (and static view), this PR was significantly faster to implement :)

#### Addresses #151 

**Special notes for your reviewer**:
- This PR is built on top of the changes that Kwasi made to update this UI to use gRPC Metadata API in favor of the deprecated REST API.

**Release note**:
```release-note
Old metadata updated with a new Lineage Explorer that provides an interactive interface to navigate through the various artifacts and executions that were invoked or created via your Kubeflow journey.
```

### Meta
/area metadata
/area front-end
/priority p0
/assign @avdaredevil
/cc @kwasi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/metadata/159)

### CODE REVIEW

File: frontend/src/pages/LineageView.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+ * limitations under the License.
+ */
+// tslint:disable: object-literal-sort-keys
+
+import * as React from 'react';
+import {Page} from './Page';
+import {classes} from 'typestyle';
+import {commonCss} from '../Css';
+import {/*Api, */ListRequest} from '../lib/Api';
+// import {MlMetadataArtifact} from '../apis/service/api';
```
Comment (kwasi) [inline frontend/src/pages/LineageView.tsx]: Can these commented lines be removed?

Comment (avdaredevil) [reply frontend/src/pages/LineageView.tsx]: These are done, PTAL.

**Pull request merged**